invenio_app_ils/config.py

```diff
@@ -793,15 +823,29 @@ def _(x):
             ),
         ),
         filters=dict(
-            document_types=terms_filter('document_types'),
-            languages=terms_filter('languages'),
+            document_types=terms_filter("document_types"),
+            languages=terms_filter("languages"),
             keywords=terms_filter("keywords.name"),
             items_available_for_loan=keyed_range_filter(
                 "circulation.items_available_for_loan",
                 {
-                    "None": {"lt": 1},
-                    "1+": {"gte": 1}
-                }
+                    "No": {"lt": 1},
+                    "Yes": {"gte": 1},
+                },
+            ),
+            number_of_items=keyed_range_filter(
+                "circulation.number_of_items",
+                {
+                    "No": {"lt": 1},
+                    "Yes": {"gte": 1},
+                },
+            ),
+            number_of_eitems=keyed_range_filter(
+                "circulation.number_of_eitems",
+                {
+                    "No": {"lt": 1},
+                    "Yes": {"gte": 1},
```

Comment:
I am sorry, nothing to do with your code, but I think how we thought the feature is not user friendly. You and @kprzerwa were probably right :)

As a user, I probably expect a filter like:
 - [ ] Books with electronic version
 - [ ] Books with copies at the library
 - [ ] Books that I can loan now

So to achieve we need to think how to do it :)